### ISSUE TITLE
Integrations by only generating docker-compose.yml

### ISSUE THREAD

**Issue by edulix** [hercules-ci/arion#145]
Is there a way to generate the `docker-compose.yml` file intead of directly running `docker-compose`? I love using nix for everything I can, but I also want to, for example, be able to use `Github Codespaces` and that requires having a `docker-compose.yml` file. If I have to duplicate the efforts by not only using arion generated one but also mantaining my own, it's less enticing.

**Comment by roberth** [comment]
Yes, you can use `arion cat` for JSON or `arion config` for YAML.

**Comment by edulix** [comment]
Awesome, thanks roberth!

**Comment by edulix** [comment]
MM any tip on how to use that efficently with devcontainer.json? Has anyone done that before?

**Comment by roberth** [comment]
Not that I'm aware. I was already using the Nix Env Selector extension instead, fwiw.

**Comment by edulix** [comment]
Interesting. I'd like to use docker-compose so that a more complex project can be tested - having multiple containers running like one for the database, another for the backend, etc. But at the same time, reusing the way I generate this for unit testing and CI and for development, all with nix ideally.

**Comment by roberth** [comment]
You could use multiple `arion-compose.nix` files (`--file`) that combine like module system imports to split the concerns of the two uses cases if necessary.
Other than that, I think I'm out of hints for now. Would be great if you could share your experience.
